### PR TITLE
Also report Cask coverage.

### PR DESCRIPTION
**Pull request by reitermarkus**
Not actually sure if this is even possible with the current setup, so I might need some help with this.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
> Not actually sure if this is even possible with the current setup, so I might need some help with this.

I think it's not; pretty sure Coveralls will only take a single test run's worth of coverage so to get this reporting again we'll need to make `brew cask-tests` part of `brew tests` and report a single coverage figure for everything.

**Comment by reitermarkus** [comment]
Nice, it's working now!

**Comment by MikeMcQuaid** [comment]
@reitermarkus Reporting both?

**Comment by reitermarkus** [comment]
Yes, just rebased so this will take a while again to show up.

**Comment by reitermarkus** [comment]
bildschirmfoto 2016-08-26 um 15 09 24

**Comment by MikeMcQuaid** [comment]
Looks like it's covering a bunch of stuff it wasn't before too 🎉

File: Library/Homebrew/test/.simplecov [hunk 1]
```diff
@@ -45,9 +44,9 @@ SimpleCov.start do
...
 
-# Don't use Coveralls outside of CI, as it will override SimpleCov's default
-# formatter causing the `index.html` not to be written once all tests finish.
-if RUBY_VERSION.split(".").first.to_i >= 2 && !ENV["HOMEBREW_INTEGRATION_TEST"] && ENV["CI"]
-  require "coveralls"
-  Coveralls.wear!
-end
+## Don't use Coveralls outside of CI, as it will override SimpleCov's default
+## formatter causing the `index.html` not to be written once all tests finish.
+#if RUBY_VERSION.split(".").first.to_i >= 2 && !ENV["HOMEBREW_INTEGRATION_TEST"] && ENV["CI"]
```
Comment (jawshooah) [inline Library/Homebrew/test/.simplecov]: Better to remove this block than leave it commented out, no?

Comment (reitermarkus) [inline Library/Homebrew/test/.simplecov]: Should already be removed in the other commit I pushed before.

**Comment by MikeMcQuaid** [comment]
Looks good to me (particularly the numbers 😍) but will let @UniqMartin give this a once over as he's worked on this coverage stuff in the past for us.

**Comment by reitermarkus** [comment]
@UniqMartin, feel free to merge this if you don't have any objections.

**Pull request merged**